kit_hunter.py

```diff
@@ -66,53 +67,53 @@
 
 print("Reading TAGS file.....")
 print("")
-f=open(phish_trk_tag_path,'r')
-phish_trk_tags_list =(f.read()).split('\n')
-if phish_trk_tags_list[-1]=="" or phish_trk_tags_list[-1]==" ":
-	phish_trk_tags_list=phish_trk_tags_list[:-1]
+f = open(phish_trk_tag_path, 'r')
+phish_trk_tags_list = (f.read()).split('\n')
+if phish_trk_tags_list[-1] == "" or phish_trk_tags_list[-1] == " ":
+    phish_trk_tags_list = phish_trk_tags_list[:-1]
 
 # - - - -
-# Some DEBUG to test out the tags file and make sure blank spaces don't bother the script.
-# It isn't needed anymore, but we're keeping it here just in case.
+# Some DEBUG to test out the tags file and make sure blank spaces don't bother
+# the script. It isn't needed anymore, but we're keeping it here just in case.
 # - - - -
-#print phish_trk_tags_list
-
+# print phish_trk_tags_list
 
 f.close()
 
 # - - - -
 # Open a new file to log the results.
 # - - - -
 
-f=open('TMP_kit_hunter_log.tmp','w') # Don't change the file name.
+f = open('TMP_kit_hunter_log.tmp', 'w')  # Don't change the file name.
 
 # - - - -
 # Here is the core.
 # Essentially, you just scan the folders and sub-folders for files that match the keywords in the tag file.
 # We're searching for TXT, PHP, .DAT, .HTM / .HTML, and .HTACCESS files.
 # - - - -
 
-data=[]
+data = []
+
+
 def phish_trk(location, phish_tag):
-   for dir_path, dirs, file_names in os.walk(location):
-      for file_name in file_names:
-         if '.txt' in file_name or '.php' in file_name or '.dat' in file_name or '.html' in file_name or '.htm' in file_name or '.htaccess' in file_name:
-            fullpath = os.path.join(dir_path, file_name)
-            for line in file(fullpath):
-               if phish_tag in line:
-                  if file_name=="tags.tag":
```

Comment:
I removed this, since we only look for files that have the particular file extension higher up. This means that "tags.tag" shouldn't ever be matched, so this condition wouldn't ever fire.